app/src/main/java/com/framgia/fsalon/screen/info/InfoUserViewModel.java

```diff
@@ -0,0 +1,107 @@
+package com.framgia.fsalon.screen.info;
+
+import android.content.Intent;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.support.v4.app.Fragment;
+import android.view.View;
+
+import com.framgia.fsalon.BR;
+import com.framgia.fsalon.screen.login.LoginActivity;
+import com.framgia.fsalon.utils.navigator.Navigator;
+
+/**
+ * Exposes the data to be used in the InfoUser screen.
+ */
+public class InfoUserViewModel extends BaseObservable implements InfoUserContract.ViewModel {
+    private InfoUserContract.Presenter mPresenter;
+    private int mProgressBarVisibility;
+    private boolean isSkipLogin;
```

Comment:
WARNING: [AvoidFieldNameMatchingMethodName] Field isSkipLogin has the same name as a method
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName